_faq/security/https-overlay.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Can I install Istio sidecar for https services?
+order: 170
+type: markdown
+---
+{% include home.html %}
+
+Yes, you can. It works both with mutual TLS enabled and disabled. Refer [here]({{home}}/docs/tasks/security/https-overlay.html) for more information.
```

Comment:
We're trying to avoid links like "here" for usability reasons, could you expand it please?

See: https://preliminary.istio.io/about/contribute/style-guide.html#create-useful-links

Reply:
Done, thanks.